### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Queues_QueueTable_Screenreader : On activating the table header the data gets sorted but the same is not announced to the screen reader users.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/AzureStorageExplorer#3507]
GitHubTags: #C&AI_Aug2020_BMAzure Tools Storage Explorer;#A11yWCAG2.1;#A11YTCS;#Azure Tools Storage Explorer;#WCAG4.1.3;#MAC;#A11ySev2;#Benchmark;#Screenreader;#VoiceOver;#FTP;

### Environment Details:
Application Name: Azure Tool Storage Explorer
Application version: 1.15.0
AzCopy Version: 10.6.0
OS Version: macOS Catalina Version 10.15.6
Screen Reader: Voiceover

### Repro Steps:
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Queues" of the selected subscription and expand it.
5. Navigate to any queue and click on it.
6. Start screen reader and navigate to table headers then activate the same to sort the data in the table. 
7. Observe if the sorting status of the data is announced or not.

### Actual:
When screen reader users navigate to the table headers and activate them the data in the table gets sorted accordingly but the same information is not conveyed to the screen reader users which is incorrect.

This issue is observed in the below scenarios:
1. Blob Container Table - Table
2. FileShares - Filesharetable
3. Disk - DiskInfoTable

### Expected:
The screen reader users should be informed about the sorting of data in the table on activating the table headers.

### User Impact:
If the screen reader users are not informed about the changes that occur on the screen then they will miss out on the information that is available for the visual users.

### Attachment:
https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-manai_microsoft_com/EaITxRNb-VlJuy-iglDY3KMBNjbBJXb2mWVBHkSdez2VGQ?e=kDXnDl

**Comment by craxal** [comment]
Duplicate of #3470.

**Comment by Ashutoshsahoo164** [comment]
GitHubTags : #Closed
